MOBILE_CLIENT/COMMON/SECURE/SECURE_REPUTATION/meta.json

```diff
@@ -0,0 +1,12 @@
+{
+    "risk_rating": "secure",
+    "short_description": "Report for domain name and IP addresses reputation score using multiple scanning APIs and databases.",
+    "references": {
+        "UrlScan" : "https://urlscan.io/",
+        "AbuseIPDB" : "https://www.abuseipdb.com/",
+        "GoogleSaveBrowsing" : "https://safebrowsing.google.com/"
+    },
+    "title": "Domain name and IP address reputation report",
+    "privacy_issue": false,
```

Comment:
this is by default false

Reply:
I dont think explicitly setting it is a bad thing, if anything its better for readability.

Reply:
> I dont think explicitly setting it is a bad thing, if anything its better for readability.

+1